packages/netlify-cms-widget-relation/src/RelationControl.js

```diff
@@ -176,40 +231,18 @@ export default class RelationControl extends React.Component {
   };
 
   loadOptions = debounce((term, callback) => {
-    const { field, query, forID, value } = this.props;
+    const { field, query, forID } = this.props;
     const collection = field.get('collection');
     const searchFields = field.get('searchFields');
     const optionsLength = field.get('optionsLength') || 20;
     const searchFieldsArray = List.isList(searchFields) ? searchFields.toJS() : [searchFields];
     const file = field.get('file');
 
-    // if the field has a previous value perform an initial search based on the value field
-    // and display it as the first option.
-    // this is required since each search is limited by optionsLength so the selected value
-    // might not show up on the first search
-    let initialSearchPromise = Promise.resolve([]);
```

Comment:
Moved to `componentDidMount` so we can keep `initialOptions` on subsequent searches.